apps/mochi-web/components/settings/notification/NotificationPage.tsx

```diff
@@ -351,7 +377,10 @@ export function NotificationPage() {
               >
                 Reset
               </ActionBarCancelButton>
-              <ActionBarConfirmButton onClick={handleSubmit(onSubmit)}>
+              <ActionBarConfirmButton
```

Comment:
The width of the button flickers when it changes to loading state. Can fix this issue by hardcoding the button’s width? Also, I believe the save bar is universal. Could you please create a share component for it?